app/assets/javascripts/common_chart_options.js

```diff
@@ -103,20 +103,22 @@ function teachersChartOptions(pointFormat) {
     },
     plotOptions: {
       series: {
-        pointWidth: 30, groupPadding: 0.075
+        pointWidth: 30, groupPadding: 0.050
```

Comment:
this wasn't quite to C's liking - there is a further in progress change for the chart columns